client/controller/index.web.js

```diff
@@ -98,6 +98,25 @@ export function clientRouter( route, ...middlewares ) {
 	page( route, ...middlewares, smartHydrate );
 }
 
+function isEditorPath( path ) {
```

Comment:
This function could use some unit tests.

Reply:
:+1: